### PR TITLE
.check_call() returns return code, and can't be used to obtain version.

### PR DESCRIPTION
**Pull request by ekoval**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+0.7.3
```
Comment (RayeN) [inline CHANGELOG.md]: Why do we have those `+` signs here?  And what about DEV first?

**Comment by RayeN** [comment]
We should remember that from now yodeploy cannot be used for python <= 2.6 (since we started to use 2.7 feature).

File: CHANGELOG.md (hunk 1 above)
Comment (RayeN) [inline CHANGELOG.md]: I'm totally dumb.  Ignore that comment.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+-----
+
+* Fix bug in `hooks.django.DjangoApp.django_version`.
```
Comment (RayeN) [inline CHANGELOG.md]: I would say that we make `manage_py` to return commands output.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by RayeN** [comment]
:+1: Looks good to me.

**Comment by RayeN** [comment]
Tests?

File: CHANGELOG.md (hunk 1 above)
Comment (adrianmoisey) [inline CHANGELOG.md]: 😆

Comment (RayeN) [inline CHANGELOG.md]: That was worth it :) (my dumb comment).  We have Adrian back \o/

**Comment by stormf** [comment]
> We should remember that from now yodeploy cannot be used for python <= 2.6 (since we started to use 2.7 feature).

That should be mentioned in the changelog. It's also a breaking api change so we should bump the major version.

**Comment by RayeN** [comment]
> That should be mentioned in the changelog. It's also a breaking api change so we should bump the major version.

Yeah :( We should have bumped the minor version (not major, just because we're still in DEV mode). But since we didn't do that - I think we should mention that in the changelog at least. Or you think we need to do another bump?

**Pull request merged**